### DISCUSSION TITLE
There are some errors when  I copy a docs' example into Visual Studio Code.

### DISCUSSION THREAD

**Discussion by zwl1619** [prisma/prisma#21903] [Q&A] [answered]
### Question

There is a section of code in the docs. 
https://www.prisma.io/docs/concepts/components/prisma-schema/data-model
![Snipaste_2023-11-11_18-38-09](https://github.com/prisma/prisma/assets/13177997/17df1557-f941-437f-9e6c-b95cb57b6ad2)

I copied the code into Visual Studio Code, there are two errors. The two error messages are the same.
![Snipaste_2023-11-11_18-47-13](https://github.com/prisma/prisma/assets/13177997/a781e3ca-0453-49ce-9f5b-d859959e3a55)

What should I do?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @zwl1619 👋 

Thank you for raising this question.

You are correct that the code section for MongoDB in that section of the docs has some errors. Please see the corrected version below:
```prisma
model User {
  id      String   @id @default(auto()) @map("_id") @db.ObjectId
  email   String   @unique
  name    String?
  role    Role     @default(USER)
  posts   Post[]
  profile Profile?
}

model Profile {
  id     String @id @default(auto()) @map("_id") @db.ObjectId
  bio    String
  user   User   @relation(fields: [userId], references: [id])
  userId String @unique @db.ObjectId
}

model Post {
  id         String     @id @default(auto()) @map("_id") @db.ObjectId
  createdAt  DateTime   @default(now())
  title      String
  published  Boolean    @default(false)
  author     User       @relation(fields: [authorId], references: [id])
  authorId   String     @db.ObjectId
  categoryIDs String[]   @db.ObjectId
  categories  Category[] @relation(fields: [categoryIDs], references: [id])
}

model Category {
  id    String @id @default(auto()) @map("_id") @db.ObjectId
  name  String
  postIDs String[] @db.ObjectId
  posts   Post[]   @relation(fields: [postIDs], references: [id])
}

enum Role {
  USER
  ADMIN
}
```

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
